src/structures/Webhook.js

```diff
@@ -213,6 +213,15 @@ class Webhook {
     return this.client.api.webhooks(this.id, this.token).delete({ reason });
   }
 
+  /**
+   * The url of this webhook
+   * @type {string}
+   * @readonly
+   */
+  get url() {
+    return `https://discordapp.com/api/webhooks/${this.id}/${this.token}`;
```

Comment:
A more library idiomatic way of doing this would be: `return this.client.options.http.api + this.client.api.webhooks[this.id][this.token];` instead of hardcoding the api base url, so that if they hypothetically ever made changes such as turning the base into `https://api.discordapp.com/webhooks` we only have to fix that URL in one location, instead of all over the library.